### ISSUE TITLE
Please release 1.7.7

### ISSUE THREAD

**Issue by ehaupt** [kometbomb/klystrack#304]
Can you please tag https://github.com/kometbomb/klystrack/commit/fe6e7465512bee32bf8e724857a7fbab902b8db0 as 1.7.7. It has been over a year since the last release and I think this revision includes some useful commits.

**Comment by LTVA1** [comment]
Hello. The developer is very busy right now and was like so last year. Consider the project to be practically abandoned for whatever amount of time.

**Comment by ehaupt** [comment]
> Hello. The developer is very busy right now and was like so last year. Consider the project to be practically abandoned for whatever amount of time.

Thanks for the heads up. I see you've got an active fork with quite a few commits ahead of the fork. I'm currently maintaining the [FreeBSD port](https://repology.org/project/klystrack/versions) of klystrack and I'm interested in a more actively maintained code base.

**Comment by LTVA1** [comment]
Yes. Although I am very stupid regarding software development, so my fork is latest commit from here + some QoL changes + a lot of garbage I personally like somehow ugly duct taped to klystrack. I don't know much about Linux-related stuff. I haven't even set up github actions to automatically compile nightly builds for every major OS.

**Comment by luzpaz** [comment]
bump

**Comment by luzpaz** [comment]
bump

**Comment by ehaupt** [comment]
There is a great [fork](https://github.com/z-erica/klystrack) by @z-erica featuring a new release https://github.com/z-erica/klystrack/releases/tag/1.7.8

This release includes a `meson.build` file, which simplifies the installation process for packagers. I have already transitioned the FreeBSD port to this fork.
